App.js

```diff
@@ -1,26 +1,100 @@
 import React from 'react'
-import styled from 'styled-components/native'
-
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
-
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
-`
-
-const App = () => {
-  return (
-    <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
-    </Container>
-  )
+import { Platform, StyleSheet, Text, View, FlatList } from 'react-native'
+import Header from './components/Header'
+import InputBar from './components/InputBar'
+import TodoItem from './components/TodoItem'
+
+export default class App extends React.Component {
+  constructor () {
+    super()
+
+    this.state = {
+      todoInput: '',
+      todos: [
+        { id: 0, title: 'Take out the trash', done: false},
+        { id: 1, title: 'Cook dinner', done: false},
+        { id: 2, title: 'Make the dishes', done: false},
+      ]
+    }
+  }
+
+  addNewTodo () {
+    let todos = this.state.todos
+    
+    todos.unshift({
+      id: todos.length + 1,
+      title: this.state.todoInput,
+      done: false
+    })
+
+    this.setState({
+      todos,
+      todoInput: ''
+    })
+  }
+  
+  toggleDone (item) {
+    let todo = this.state.todos
+
+    todos = todos.map((todo) => {
+      if (todo.id == item.id) {
+        todo.done = !todo.done
+      }
+
+      return todo
+    })
+
+    this.setState({todos})
+  }
```

Comment:
You have different naming here (todo vs todos), so the app is crashing... Change so that it corresponds, and it should work